stylix/helper.nix

```diff
@@ -0,0 +1,7 @@
+{
+  # Turns a list of fonts into a list of font names
+  fontNames = builtins.map ({name, ...}: name);
+
+  # Turns a list of fonts into a list of font packages
+  fontPackages = builtins.map ({package, ...}: package);
+}
```

Comment:
These could be simplified to `lib.catAttrs "name"` and `lib.catAttrs "package"` respectively. IMO that's simple enough that we wouldn't need them defined as a function